plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1482,23 +1482,23 @@ public void testNullBehavior()
         assertThat(query("""
                 SELECT COUNT(*)
                 FROM alltypes
-                WHERE string_col IS NULL"""))
-                .matches("VALUES (BIGINT '0')")
-                .isNotFullyPushedDown(FilterNode.class);
+                WHERE string_col IS NULL OR string_col = 'null'"""))
+                .matches("VALUES (BIGINT '1')")
```

Comment:
This table has 'null' string in string_col, so pinot will return 1 row after pushing down.

Reply:
ah, I see, got it!